### PR TITLE
Add dnn parameter in config file

### PR DESCRIPTION
**Pull request by orion-belt**

### CODE REVIEW

**Comment by thakurajayL** [comment]
ok to test

File: config/gnbsim.yaml [hunk 1]
```diff
@@ -68,6 +69,7 @@ configuration:
       opc: "981d464c7c52eb6e5036234984ad0bcf"
       key: "5122250214c33e723a5dd523fc145fc0"
       sequenceNumber: "16f3b3f70fc2"
+      dnn: "internet"
```
Comment (thakurajayL) [inline config/gnbsim.yaml:72]: Can we make it list ? just a thought. We would be supporting multiple PDU sessions at some point in time. or should we keep it simple for now ?

**Comment by thakurajayL** [comment]
Do you want to update NAS module tag in the gnbsim ? I just created tag for gas module v1.1.3 with PTI fix.

File: config/gnbsim.yaml (hunk 1 above)
Comment (orion-belt) [reply config/gnbsim.yaml:72]: Can we keep it simple now? We need to map dnnList with SliceSupportList for multiple PDU sessions for each UE.
I will try to create PR for that later.

**Comment by orion-belt** [comment]
> Do you want to update NAS module tag in the gnbsim ? I just created tag for gas module v1.1.3 with PTI fix.

Addressed in latest commit

File: config/gnbsim.yaml (hunk 1 above)
Comment (orion-belt) [reply config/gnbsim.yaml:72]: Also, please make a note dnn and snssai is not list when it is added in `PduSessionEstablishmentRequest`.

File: config/gnbsim.yaml [hunk 2]
```diff
@@ -40,6 +40,9 @@ configuration:
               taiSliceSupportList:
                 - sst: 1 # Slice/Service Type (uinteger, range: 0~255)
                   sd: 010203 # Slice Differentiator (3 bytes hex string, range: 000000~FFFFFF)
+                - sst: 1
+                  sd: 000001
+                - sst: 2
```
Comment (thakurajayL) [inline config/gnbsim.yaml:45]: What happens if there is no sd ?

Comment (orion-belt) [reply config/gnbsim.yaml:45]: It is not encoded in the NGSetupRequest message by Gnbsim.

**Review by thakurajayL — APPROVED** [review]

File: config/gnbsim.yaml (hunk 2 above)
Comment (thakurajayL) [reply config/gnbsim.yaml:45]: Could you please point me to code where its skipped ? Approving code, assuming we will not be dealing with situation immediately.

Comment (orion-belt) [reply config/gnbsim.yaml:45]: [Optional field Sd decoding at NGAP ](https://github.com/omec-project/ngap/blob/main/ngapConvert/SNssai.go#L27)

**Pull request merged**